drivers/si70xx/si70xx.c

```diff
@@ -18,22 +18,25 @@
  * @}
  */
 
-
+#include <string.h>
 #include "xtimer.h"
 
 #include "si70xx.h"
 
+#define SI70XX_I2C     (dev->params.i2c_dev)
```

Comment:
I know, but I think that prefixing gives more security since BUS and ADDR are too general names that could be used elsewhere.